app/main.ipynb

```diff
@@ -126,19 +158,32 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 10,
-   "outputs": [],
    "source": [
     "array = np.array([2.3, 7.8, 3.2, 1.1, 5.8, 9.5, 17.6, 11.1])\n",
-    "# write your code here"
+    "selected_elements = array[[1, 2, 4, 5]]\n",
+    "result = np.sum(selected_elements)\n",
```

Comment:
The selected elements should sum to 27, but the current selection results in 26.3. Check the indices used for selection to ensure they match the task requirements.